spec/facter/resolvers/windows/networking_spec.rb

```diff
@@ -249,6 +260,14 @@
       it 'returns domain' do
         expect(resolver.resolve(:domain)).to eql(domain)
       end
+
+      it 'replaces invalid characters in the friendly name' do
+        allow(friendly_name_ptr).to receive(:get_bytes).with(0, 2).and_return(+"\xf0\xdc")
+        allow(friendly_name_ptr).to receive(:get_bytes).with(2, 2).and_return(+"\0\0")
```

Comment:
Should be able to replace both lines above with a single call to:

```suggestion
        stub_utf16le_bytes(friendly_name_ptr, "\xf0\xdc")
```

You won't have to worry about the frozen string since `stub_utf16le_bytes` calls `str.encode(Encoding::UTF_16LE)` which makes a copy. Also it handles adding the null terminator.